woocommerce/wc-admin
# Remove googlefonts/material icons. Migrate to @wordpress/icons and gridicons

Fixes #4472

There is lots of good rationale for dropping google fonts / material icons. See the original issue for details.

I have tried to first and foremost use `@wordpress/icons`, but if there was nothing similar there I have used gridicons. 

I'll post some screenshots of the different areas I've updated because some are departures.

cc @elizaan36 could you take a look at this?

### Screenshots

#### The product management setup task (probably the biggest departures are here):

<img width="793" alt="Screenshot 2021-02-16 at 3 07 05 PM" src="https://user-images.githubusercontent.com/1281828/108009948-a9a4ed80-7068-11eb-9b8b-094e2dd53800.png">

Here is the **old** for that one:

<img width="713" alt="Screenshot 2021-02-16 at 2 28 20 PM" src="https://user-images.githubusercontent.com/1281828/108007687-757afe00-7063-11eb-8570-8876be8f8e54.png">

#### The theme preview:

<img width="260" alt="Screenshot 2021-02-16 at 2 12 48 PM" src="https://user-images.githubusercontent.com/1281828/108007716-8461b080-7063-11eb-81a2-32c9ae6a2e60.png">

This one looks almost identical to the original.


#### The analytics search box:

<img width="333" alt="Screenshot 2021-02-16 at 2 25 28 PM" src="https://user-images.githubusercontent.com/1281828/108007880-de627600-7063-11eb-80f3-3baf16759388.png">


#### One of the "business details" a/b tests:

<img width="651" alt="Screenshot 2021-02-16 at 2 51 57 PM" src="https://user-images.githubusercontent.com/1281828/108009079-c17b7200-7066-11eb-8124-ed48503c180e.png">


### Detailed test instructions:

1. Go to the product management setup task and see how the icons look. the chevrons, the icons on the left hand side etc (see the screenshot above)

2. Preview a theme. To do this, go through the OBW to the theme step, then click the "live demo" button. You should see the device icons in the top right of the screen.

3. See how the search icon looks in the select control. Go to `Analytics > Coupons`. Inspect the search box at bottom of screen. (See the screenshot above for reference.)

4. For the business details a/b test go through the OBW and make sure to choose US as location and industry of fashion or health and beauty, once you get to the business details step, there is a popover that appears if you hover over the info icon. Confirm the info icon and check marks in the popover look good.
